packages/fxa-auth-server/lib/payments/stripe.js

```diff
@@ -809,7 +787,10 @@ class StripeHelper {
         failure_message = /** @type {Charge} */ (charge).failure_message;
       }
 
-      const planName = await this.formatPlanDisplayName(sub.plan);
+      const product = await this.expandResource(sub.plan.product, 'Product');
```

Comment:
Should this be wrapped in try/catch in case the resource fails to be found for any reason?

Reply:
If this resource fails to be found, we actually cannot display the information to the user - since we rely on the Product name. So, I think that this should result in an error. If we were to simply exclude Subscriptions from the response that we were unable to load, then we would be in a situation where the GUI is misleading the customer. Also, this situation would only occur in the event that Stripe is down, in which case, the user would be unable to manage their subscriptions at all.